core/native-bridge.js

```diff
@@ -0,0 +1,563 @@
+      );
+      if (result.success === false) {
+        c.error(result.error);
+      } else {
+        c.dir(result.data);
+      }
+      c.groupEnd();
+    } else {
+      if (result.success === false) {
+        c.error('LOG FROM NATIVE', result.error);
+      } else {
+        c.log('LOG FROM NATIVE', result.data);
+      }
+    }
+  };
+
+  const createLogToNative = c => call => {
+    if (isFullConsole(c)) {
+      c.groupCollapsed(
+        '%cnative %c' +
+          call.pluginId +
+          '.' +
+          call.methodName +
+          ' (#' +
+          call.callbackId +
+          ')',
+        'font-weight: lighter; color: gray',
+        'font-weight: bold; color: #000',
+      );
+      c.dir(call);
+      c.groupEnd();
+    } else {
+      c.log('LOG TO NATIVE: ', call);
+    }
+  };
+
+  const isFullConsole = c => {
+    if (!c) {
+      return false;
+    }
+
+    return (
+      typeof c.groupCollapsed === 'function' ||
+      typeof c.groupEnd === 'function' ||
+      typeof c.dir === 'function'
+    );
+  };
+
+  const serializeConsoleMessage = msg => {
+    if (typeof msg === 'object') {
+      try {
+        msg = JSON.stringify(msg);
+      } catch (e) {
+        // ignore
+      }
+    }
+
+    return String(msg);
+  };
+
+  // patch window.console on iOS and store original console fns
+  const isIos = getPlatformId(win) === 'ios';
+  const originalConsole = { ...win.console };
+
+  if (win.console && isIos) {
+    for (const logfn of BRIDGED_CONSOLE_METHODS) {
+      win.console[logfn] = (...args) => {
+        const msgs = [...args];
+
+        originalConsole[logfn](...msgs);
+
+        try {
+          cap.toNative('Console', 'log', {
+            level: logfn,
+            message: msgs.map(serializeConsoleMessage).join(' '),
+          });
+        } catch (e) {
+          // error converting/posting console messages
+          originalConsole.error(e);
+        }
+      };
+    }
+  }
+
+  cap.logJs = (msg, level) => {
+    switch (level) {
+      case 'error':
+        win.console.error(msg);
+        break;
+      case 'warn':
+        win.console.warn(msg);
+        break;
+      case 'info':
+        win.console.info(msg);
+        break;
+      default:
+        win.console.log(msg);
+    }
+  };
+
+  cap.logToNative = createLogToNative(win.console);
+  cap.logFromNative = createLogFromNative(win.console);
+
+  cap.handleError = err => win.console.error(err);
+
+  win.Capacitor = cap;
+};
+
+function initBridge(win) {
+  const cap = win.Capacitor || {};
+
+  // keep a collection of callbacks for native response data
+  const callbacks = new Map();
+
+  // Counter of callback ids, randomized to avoid
+  // any issues during reloads if a call comes back with
+  // an existing callback id from an old session
+  let callbackIdCount = Math.floor(Math.random() * 134217728);
+
+  let postToNative = null;
+
+  const isNativePlatform = () => getPlatformId(win) !== 'web';
```

Comment:
```suggestion
  const isNativePlatform = () => true;
```

native-bridge.js is executed in native apps only, so no need to check the platform, will always be true